scss/mixins/_mixins.scss

```diff
@@ -13,6 +13,14 @@
   }
 }
 
+@mixin highlight-tab(){
+  box-shadow: inset 0 0 0 2px $blue-light;
```

Comment:
Should we remove the `inset` part so that the highlight is larger? The customer complained about the visibility of the focused style, so I think it's better to make it bigger.